### DISCUSSION TITLE
Exploring Cumulants Expansion for Mean-Field Simulation of Large Open Quantum Systems

### DISCUSSION THREAD

**Discussion by Papageno2** [qutip/qutip#2213] [Ideas]
Hello QuTiP team,

I wanted to inquire about the possibility of a counterpart to the Julia package `QuantumCumlants.jl` in QuTiP.

Especially when simulating large open systems, such as the open Dicke model, I've found that `qutip.piqs` greatly facilitates model building and simulation through its support for (stochastic) master equation solvers.

Looking ahead, I am curious if there are any plans for QuTiP to delve into (stochastic) mean-field equations through the utilization of cumulant equations.

Thank you for your efforts in advancing the field of quantum simulation.

**Comment by hodgestar** [comment]
Hi Papageno,

If you have a concrete proposal for implementing this in QuTiP, we would
love to hear it. It definitely falls within QuTiP's scope of solving
evolution equations for open quantum systems, so it's mostly a matter of
practicalities.

Regards,
Simon
